pkg/catalogv2/helm/helm2.go

```diff
@@ -49,6 +55,7 @@ func fromHelm2Data(data string, isNamespaced IsNamespaced) (*v1.ReleaseSpec, err
 	return fromHelm2ReleaseToRelease(release, isNamespaced)
 }
 
+// toTime receives timestamp pb and returns the corresponding metav1.Time struct
```

Comment:
// toTime receives timestamp google protobuf format and returns the corresponding metav1.Time struct